src/main/java/seedu/duke/commands/task/TaskCommand.java

```diff
@@ -1,29 +1,37 @@
 package seedu.duke.commands.task;
 
+import seedu.duke.DukeExceptions;
 import seedu.duke.commands.Command;
 import seedu.duke.project.Project;
-import seedu.duke.project.ProjectList;
 import seedu.duke.task.Task;
 import seedu.duke.ui.Ui;
 
-import java.time.LocalDate;
 import java.util.ArrayList;
+import java.util.HashMap;
+
+import static seedu.duke.Parser.getHashValue;
 
 public class TaskCommand extends Command {
 
-    private String description;
     private int projectIndex;
+    private String description;
+    HashMap<String, String> params;
 
-    public TaskCommand(String description, int projectIndex) {
-        this.description = description;
+    public TaskCommand(HashMap<String, String> params, int projectIndex) throws DukeExceptions {
+        this.params = params;
         this.projectIndex = projectIndex;
+        this.parse();
+    }
+
+    public void parse() throws DukeExceptions {
+        this.description = getHashValue(params, "n");
     }
```

Comment:
For future reference, should we remove this as the task does not require multiple inputs.